shared/oae/css/oae.components.css

```diff
@@ -936,6 +936,15 @@ div.oae-thumbnail i.fa {
 
 /* Left-hand navigation collapsed view */
 @media (max-width: 991px) {
+    /* hide from screen readers */
+    .oae-lhnavigation {
+        display: none;
```

Comment:
Actually, scrap the `hide` class as we only want this to happen on <991px devices. However, the `display:none` should still only be applied after the animation has finished.